test/e2e/priority_scheduling/priority_scheduling_test.go

```diff
@@ -251,6 +252,217 @@ var _ = ginkgo.Describe("PriorityScheduling", func() {
+			RestartPolicy:     v1.RestartPolicyNever,
+		}
+
+		normalPodConf = k8s.TestPodConfig{
+			Name: "test-normal-priority-" + common.RandSeq(5),
+			Labels: map[string]string{
+				constants.LabelQueueName:     "root.default",
+				constants.LabelApplicationID: "app-normal-" + common.RandSeq(5)},
+			Namespace: ns,
+			Annotations: &k8s.PodAnnotation{
+				TaskGroupName: "group-normal",
+				TaskGroups: []interfaces.TaskGroup{
+					{
+						Name:        "group-normal",
+						MinMember:   int32(1),
+						MinResource: taskGroupMinResource,
+					},
+				},
+			},
+			Resources:         rr,
+			PriorityClassName: normalPriorityClass.Name,
+			RestartPolicy:     v1.RestartPolicyNever,
+		}
+
+		highPodConf = k8s.TestPodConfig{
+			Name: "test-high-priority-" + common.RandSeq(5),
+			Labels: map[string]string{
+				constants.LabelQueueName:     "root.default",
+				constants.LabelApplicationID: "app-high-" + common.RandSeq(5)},
+			Namespace: ns,
+			Annotations: &k8s.PodAnnotation{
+				TaskGroupName: "group-high",
+				TaskGroups: []interfaces.TaskGroup{
+					{
+						Name:        "group-high",
+						MinMember:   int32(1),
+						MinResource: taskGroupMinResource,
+					},
+				},
+			},
+			Resources:         rr,
+			PriorityClassName: highPriorityClass.Name,
+			RestartPolicy:     v1.RestartPolicyNever,
+		}
+
+		var sleepPod, lowPod, normalPod, highPod *v1.Pod
+		By("Create sleep pod to consume queue")
+		sleepPod, err = k8s.InitTestPod(sleepPodConf)
+		Ω(err).NotTo(gomega.HaveOccurred())
+		sleepPod, err = kubeClient.CreatePod(sleepPod, ns)
+		Ω(err).NotTo(gomega.HaveOccurred())
+		err = kubeClient.WaitForPodRunning(ns, sleepPod.Name, 1*time.Minute)
+		Ω(err).NotTo(gomega.HaveOccurred())
+
+		By("Submit low priority job")
+		lowPod, err = k8s.InitTestPod(lowPodConf)
+		Ω(err).NotTo(gomega.HaveOccurred())
+		lowJob := k8s.InitTestJob(lowPod.Labels[constants.LabelApplicationID], 1, 1, lowPod)
+		lowJob, err = kubeClient.CreateJob(lowJob, ns)
+		Ω(err).NotTo(gomega.HaveOccurred())
+		err = kubeClient.WaitForJobPodsCreated(ns, lowJob.Name, 1, 30*time.Second)
+		Ω(err).NotTo(gomega.HaveOccurred())
+
+		By("Submit normal priority job")
+		normalPod, err = k8s.InitTestPod(normalPodConf)
+		Ω(err).NotTo(gomega.HaveOccurred())
+		normalJob := k8s.InitTestJob(normalPod.Labels[constants.LabelApplicationID], 1, 1, normalPod)
+		normalJob, err = kubeClient.CreateJob(normalJob, ns)
+		Ω(err).NotTo(gomega.HaveOccurred())
+		err = kubeClient.WaitForJobPodsCreated(ns, normalJob.Name, 1, 30*time.Second)
+		Ω(err).NotTo(gomega.HaveOccurred())
+
+		By("Submit high priority job")
+		highPod, err = k8s.InitTestPod(highPodConf)
+		Ω(err).NotTo(gomega.HaveOccurred())
+		highJob := k8s.InitTestJob(highPod.Labels[constants.LabelApplicationID], 1, 1, highPod)
+		highJob, err = kubeClient.CreateJob(highJob, ns)
+		Ω(err).NotTo(gomega.HaveOccurred())
+		err = kubeClient.WaitForJobPodsCreated(ns, highJob.Name, 1, 30*time.Second)
+		Ω(err).NotTo(gomega.HaveOccurred())
```

Comment:
Can we submit the jobs in reverse order?

Reply:
I would like to submit from low priority job, so we can test the system consider priority first and then submission time.